src/lib/stitching/vortex/stitching.ts

```diff
@@ -63,21 +63,36 @@ export const vortexStitchingEnvironment = () => ({
               _id
             }
             category
+            is_for_sale
             is_price_hidden
+            is_in_auction
+            price_currency
+            artists {
+              _id
+            }
           }
         `,
         resolve: async (source, _, context, info) => {
           const {
-            priceCents,
-            widthCm,
-            heightCm,
             artist,
+            artists,
             category,
+            heightCm,
+            is_for_sale,
+            is_in_auction,
             is_price_hidden,
+            price_currency,
+            priceCents,
+            widthCm,
           } = source
           // fail if we don't have enough info to request a histogram
+
           if (
             is_price_hidden ||
+            is_in_auction ||
+            price_currency !== "USD" ||
+            (artists && artists.length > 1) ||
+            !is_for_sale ||
```

Comment:
The lab feature check in this resolver might be able to be removed btw. This was accomplished in https://github.com/artsy/reaction/pull/2240

Reply:
Good point! We already have a ticket for disabling the lab feature in Jira so I merged this now and we can follow up with another PR if we decide to do it early.

Reply:
Oh! Shucks, missed that. Good catch @mzikherman!